model/checkininfo.go

```diff
@@ -0,0 +1,11 @@
+package model
+
+// CheckInInfoWebConnectivity contains the array of URLs returned by the checkin API
+type CheckInInfoWebConnectivity struct {
+	URLs []URLInfo `json:"urls"`
+}
+
+// CheckInInfo contains the return test objects from the checkin API
+type CheckInInfo struct {
+	WebConnectivity CheckInInfoWebConnectivity `json:"web_connectivity"`
```

Comment:
My understanding is that this property is optional. If that is the case, then we should use instead:

```suggestion
	WebConnectivity *CheckInInfoWebConnectivity `json:"web_connectivity"`
```

This also means that we need to be careful when manipulating the returned value. This is a sad side effect of languages that allow for `nil` values and we need to live with that.